packages/dashboard/src/pages/Settings/api.ts

```diff
@@ -3,6 +3,7 @@ import { Messages, Services } from '../../API'
 import type { DataProvider } from '@masknet/public-api'
 import type { AccountValidationType, BackupFileInfo } from './type'
 export const [useLanguage] = createGlobalState(Services.Settings.getLanguage, Messages.events.languageSettings.on)
+export const [useAvatar] = createGlobalState(Services.Settings.getAvatar, Messages.events.avatarSettings.on)
```

Comment:
Well, this doesn't look right to me. Avatar should be derived from the current persona, why it should be store in settings?